### ISSUE TITLE
ChipUtil fails to initialize on Paper `testing/no-relocation`

### ISSUE THREAD

**Issue by datatags** [GamerCoder215/MobChip#112]
### Bug Type

Compatibility Issues, Console Errors

### MobChip Version

1.9.4-SNAPSHOT

### Server Version

Paper `testing/no-relocation`: https://github.com/PaperMC/testing/releases

### Description

ChipUtil fails to initialize when used with this Paper build.

### Expected Behavior

ChipUtil should initialize correctly and pick the most correct module to use for the server.

### Actual Behavior

This error is printed to the console (in this case, MobChip is shaded into my plugin):
```
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at be.isach.ultracosmetics.shaded.mobchip.abstraction.ChipUtil.getServerVersion(ChipUtil.java:182) ~[?:?]
	at be.isach.ultracosmetics.shaded.mobchip.abstraction.ChipUtil.getWrapper(ChipUtil.java:186) ~[?:?]
	at be.isach.ultracosmetics.shaded.mobchip.bukkit.BukkitBrain.<clinit>(BukkitBrain.java:45) ~[?:?]
	at be.isach.ultracosmetics.cosmetics.pets.Pet.clearPathfinders(Pet.java:128) ~[?:?]
	at be.isach.ultracosmetics.cosmetics.pets.Pet.initializeEntity(Pet.java:113) ~[?:?]
	at be.isach.ultracosmetics.cosmetics.pets.Pet.onEquip(Pet.java:89) ~[?:?]
	at be.isach.ultracosmetics.cosmetics.Cosmetic.equip(Cosmetic.java:106) ~[?:?]
	at be.isach.ultracosmetics.cosmetics.type.CosmeticType.equip(CosmeticType.java:173) ~[?:?]
	at be.isach.ultracosmetics.cosmetics.type.PetType.equip(PetType.java:66) ~[?:?]
	at be.isach.ultracosmetics.cosmetics.type.PetType.equip(PetType.java:31) ~[?:?]
	at be.isach.ultracosmetics.command.subcommands.SubCommandToggle.toggle(SubCommandToggle.java:92) ~[?:?]
	at be.isach.ultracosmetics.command.subcommands.SubCommandToggle.onExeAnyone(SubCommandToggle.java:58) ~[?:?]
	at be.isach.ultracosmetics.command.CommandManager.onCommand(CommandManager.java:87) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	... 9 more
```

### Additional Information

https://forums.papermc.io/threads/paper-velocity-1-20-4.998/#post-2955
https://github.com/UltraCosmetics/UltraCosmetics/issues/153

I'd be willing to attempt a PR for this if you're busy

**Comment by GamerCoder215** [comment]
Thanks for the heads-up. Will implement a fix along with other issues in v1.9.5.

**Comment by GamerCoder215** [comment]
Added a draft in 58d6151. I kinda rushed through it by making a simple `switch` tree and haven't tested it. Once I finish the rest of the bugs targeted for v1.9.5, it should be released.
